test/presenters/corporate_information_page_presenter_test.rb

```diff
@@ -14,6 +14,11 @@ def format_name
       assert_equal "About us - Department of Health", presented_item.page_title
     end
 
+    test 'does not present an organisation in the title when it is not present in links' do
+      presented_item = presented_item(format_name, "links" => { "organisations" => [] })
```

Comment:
I think to fully test the current bug we're seeing, the `links` hash shouldn't even have the organisations key/value present... I think your code implementation would still work, but would good to update this test to be more explicit about the exact conditions under which the bug happens.